src/components/shared/CardList/Card/Card.jsx

```diff
@@ -9,26 +9,55 @@ export class Card extends React.Component {
 
     constructor() {
         super();
-        this.state = {};
+        this.state = {
+            flipped: false
+        };
         this.handleClick = this.handleClick.bind(this);
     }
 
+    componentDidMount() {
+        if (!this.props.myCards) {
+            setTimeout(() => {            
+                this.setState({
+                    flipped: true
+                })
+            }, 300);
+        }
+    }
+    
     handleClick() {
         if (this.props.enabled) this.props.handleClick(this.props.card.cardId);
     }
 
-    render() {
-        return (
-            <div data-cy='card-wrapper' className={cx(styles.cardWrapper, { enabled: this.props.enabled }, { disabled: !this.props.enabled }, {selected : this.props.playedCardId === this.props.card.cardId})} onClick={this.handleClick}>
-                <img data-cy='card' className={styles.card} alt={`card-${this.props.card.cardId}`} src={require(`../../../../images/cards/card (${this.props.card.cardId}).jpg`)} />
-                {(this.props.card.votes !== undefined) && <div className={styles.vote} data-cy='vote'>Votes: {this.props.card.votes}</div>}
-            </div>
-        );
+    render() {        
+        if (this.props.myCards) {
```

Comment:
Not sure about this `if else` statement, is there some way you can generalise it?